lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_helper.rb

```diff
@@ -4,6 +4,19 @@ module Fastlane
   UI = FastlaneCore::UI unless Fastlane.const_defined?("UI")
   module Helper
     module FirebaseAppDistributionHelper
+      APK = 'APK'
+      AAB = 'AAB'
+      IPA = 'IPA'
```

Comment:
I think it would be more idiomatic to represent these as symbols instead of strings. You could even do away with these variables and just use `:apk`, `:aab`, and `:ipa` throughout, like we do with the platforms. The only limitation I see is where we insert these into messages to display to the user, but maybe we could perform the translation from `:apk` -> `"APK"` in that one case?

Reply:
I like this idea. What about an uppercase symbol?!

Reply:
I feel like I normally see them as lowercase? But sure why not!

Reply:
rubocop didn't yell at me so going with it 😆